src/ConfigHandler.cpp

```diff
@@ -23,16 +23,38 @@ void ConfigHandler::setConfigFilePath(string filepath) {
   mConfigFilePath = filepath;
 }
 
-TResult<string> ConfigHandler::getValue(string section, string key) {
-  bool a_bIsUtf8 = false;        // use OS native encoding
-  bool a_bUseMultiKey = false;   // don't support duplicated keys
-  bool a_bUseMultiLine = false;  // don't support multiline values for a key
+TResult<string> ConfigHandler::getValueString(string section, string key) {
+  bool const isUtf8 = false;        // use OS native encoding
+  bool const useMultiKey = false;   // don't support duplicated keys
+  bool const useMultiLine = false;  // don't support multiline values for a key
 
-  CSimpleIniA ini(a_bIsUtf8, a_bUseMultiKey, a_bUseMultiLine);
+  CSimpleIniA ini(isUtf8, useMultiKey, useMultiLine);
   SI_Error rc = ini.LoadFile(mConfigFilePath.c_str());
   if (rc < 0)
     return Error(ErrorCode::FileNotFound,
                  "ConfigHandler.getValue: couldn't load file");
 
   return ini.GetValue(section.c_str(), key.c_str());
 }
+
+TResult<int> ConfigHandler::getValueInt(string section, string key) {
```

Comment:
i just saw there exists a `GetLongValue` in `SimpleIni`

Reply:
thanks :)   implemented in https://github.com/skaupper/VirtualJukebox/pull/19/commits/224b099a4dd424e20e13e46e34e7ff2004a94717